recipe/meta.yaml

```diff
@@ -28,7 +28,9 @@ requirements:
   run:
     - python
     - dataclasses >=0.6  # [py<37]
-    - typing-extensions >=3.7.4.3
+    # optional deps
+    - email-validator >=1.0.3
```

Comment:
Putting` email-validator` in run makes it required rather than optional.
Instead you could put it under `run_constrained`. If the user decides to install email-validator, this will check that version is 1.0.3 or above.
```
  run_constrained:
    - email-validator >=1.0.3
```

Reply:
Thank you! I wasn't sure how to do this properly.